src/app/task/task.component.ts

```diff
@@ -0,0 +1,5 @@
+export interface Task {
+  title: string;
+   done: boolean; 
+   date: Date;
+}
```

Comment:
Po pierwsze to nie jest komponent. Po drugie plik jest źle sformatowany

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/3aa1f288e2b2235bebdb49335fcc41f9709ff33b